src/kernel/arch/x86/arch.zig

```diff
@@ -1,63 +1,63 @@
 // Zig version: 0.4.0
 
+const std = @import("std");
 const builtin = @import("builtin");
 const gdt = @import("gdt.zig");
 const idt = @import("idt.zig");
 const irq = @import("irq.zig");
 const isr = @import("isr.zig");
 const log = @import("../../log.zig");
 const pit = @import("pit.zig");
+const paging = @import("paging.zig");
+const MemProfile = @import("../../mem.zig").MemProfile;
 
 pub const InterruptContext = struct {
     // Extra segments
     gs: u32,
     fs: u32,
     es: u32,
     ds: u32,
-
+    
```

Comment:
Got some whitespace before the comments in this struct

Reply:
Alright, I'll remove whitespaces